### ISSUE TITLE
Relative time in English has different casing from CLDR

### ISSUE THREAD

**Issue by sandstrom** [twitter/twitter-cldr-rb#187]
I've noticed that relative time has different casing (initials upper-cased) from Unicode CLDR.

``` ruby
TwitterCldr::Localized::LocalizedTimespan.new(3600 * 24 * 2, :locale => 'en').to_s
# => "In 2 days" (expected "in 2 days")
```

These values are lower-case:
https://github.com/unicode-cldr/cldr-dates-full/blob/master/main/en/dateFields.json#L184

These values have first-letter capitalized:
https://github.com/twitter/twitter-cldr-rb/blob/master/resources/locales/en/units.yml#L11

Is this an issue with the imports, or something else?

**Comment by camertron** [comment]
Hey @sandstrom, unfortunately the units schema in the CLDR changed after v23 and we never put in the effort to refactor. This also means we're stuck using the [data from v23](http://unicode.org/cldr/trac/browser/tags/release-23/common/main/en.xml#L4771), which contains those upper-cased "In" bits. I would love to get all parts of TwitterCLDR using the latest CLDR data (v29 I believe), but my time is limited these days. I would however welcome a pull request if you've got the time to dive in.

**Comment by sandstrom** [comment]
@camertron I appreciate the update! I'm also short on time the next 2 weeks. Perhaps I can get around to it later, but I cannot promise anything.

Anyhow, thanks for a great library! We're a tiny startup and this is very helpful to us! It's great that you and Twitter will make this OS contribution ⛵

**Comment by camertron** [comment]
Glad to hear you're getting some mileage out of the library! Can I ask what your startup is and what you're using TwitterCLDR for? I ask for my own curiosity :)

One thing I forgot to mention is that TwitterCLDR does have the ability to downcase text in just about any script. You can either call `#downcase` on an instance of `LocalizedString` or use the caser directly:

```ruby
"ABC".localize.downcase.to_s  # => "abc"
TwitterCldr::Shared::Caser.downcase("ABC")  # => "abc"
```

For these examples you could of course have used ruby's built-in `String#downcase`, but the TwitterCLDR caser should work for many other scripts and languages as well.

**Comment by sandstrom** [comment]
It's an finance/employee tool, sold to companies. We're only a couple of people. Most localization occur in the JS-layer (SPA), but some things also need localization at the server. Email bodies and some other server-generated content. For this TwitterCLDR is great!

Great suggestion, that's sort of what I did.

You asked about [renaming the project](https://github.com/twitter/twitter-cldr-rb/issues/162) previously. How about *vernacular*? (*rosetta* would have been great, but there is a gem with that name already; on *ricu* I think it's bland and difficult to remember)

**Comment by camertron** [comment]
Cool thanks for sharing!

I kinda like _vernacular_, although we decided a while ago that renaming the project wouldn't really benefit it in any way. A name is just a name.

I have always wanted to integrate TwitterCLDR into Rails and libraries like Globalize. It's really just time constraints that have prevented it. But, viva la open-source! Hopefully someone with more time/resources can take up that particular mantle.

**Comment by shlok007** [comment]
@cameartron that sounds cool. I really wish to implement that. Can you please elaborate a little more? :smiley:

**Comment by camertron** [comment]
@shlok007 Awesome, let me know what you create!

**Comment by camertron** [comment]
This should be fixed with the release of v4.0.0.
